capa/main.py

```diff
@@ -1087,7 +1087,21 @@ def handle_common_args(args):
         args.signatures = sigs_path
 
 
+def last_resort_exception_handler(exctype: Exception, value: str, traceback: traceback) -> None:   
+    if SHOW_STACK_TRACE:
```

Comment:
i wonder if you can do this without the global variable, which we try pretty hard to avoid.